### DISCUSSION TITLE
How to change the resolution of the mpegts output

### DISCUSSION THREAD

**Discussion by kabadisha** [blakeblackshear/frigate#4799] [Ask A Question] [answered]
Hey Guys & Girls,
Hopefully, someone can point me in the right direction.

For each camera I operate, I see two ffmpeg processes running:
```
00:00:24 ffmpeg -hide_banner -loglevel warning -c:v h264_cuvid -i rtsp://loungecam.lan:554/h264 -r 30 -s 1640x1232 -f rawvideo -pix_fmt yuv420p pipe:
00:00:29 ffmpeg -f rawvideo -pix_fmt yuv420p -video_size 1640x1232 -i pipe: -f mpegts -s 958x720 -codec:v mpeg1video -q 8 -bf 0 pipe:
```
The second one seems to be converting to an mpegts stream at 958x720. Can anyone tell me what this mpegts stream is for and how I can change it to the native resolution of the camera (1640x1232).
My guess is that it's for the Frigate WebUI, but my Google sleuthing can't get me any further.

Many thanks

**Comment by NickM-27** [comment]
It's for the frigate webUI and it is configurable using the live -> height tag but is called as the resolution of the detect stream (which we DONT RECOMMEND setting to full stream)

In the next version of frigate the live stream will be able to use the main stream without affecting detect.

**Reply by kabadisha** [reply]
Thanks. Could I ask the reason that you don't recommend setting the detect resolution to the native stream res?

On my system, I found that it actually reduced system resource usage. Specifically, it dropped CPU usage by 5-10%.

I'm using my Nvidia P600 for video decoding and yolo4 detection.

**Reply by NickM-27** [reply]
Did you use your detect -> width / height accordingly? Whatever those values are, frigate resizes the detect stream to that

We don't recommend because running motion detection on 4K is pointless in most cases, the regions are downsized to the models trained size (320x320 by default) so there's no benefit and it uses more cpu

**Comment by blakeblackshear** [comment, marked as answer]
It's the live view for the camera using jsmpeg. It's configurable under the `live` config option in the config.

**Reply by kabadisha** [reply]
Super. Thanks guys - I'll have a play with it when I've got a moment.

I have a feeling that the re-sizing is adding a little CPU load that I can avoid. Trying to keep energy consumption down.
Probably a minor difference, but every little helps.
